keg_mail/content.py

```diff
@@ -124,6 +127,23 @@ def __init__(self, subject, content, type_=None):
         self.content = content
         self.type_ = type_
 
+    @classmethod
+    def validate_address(cls, address):
+        """Returns `True` if the address looks like a valid e-mail address"""
+        try:
+            validator_collection.email(address)
+            return True
+
+        except ValueError:
+            return False
+
+    @classmethod
+    def address_might_be_invalid(cls, address):
+        if not cls.validate_address(address):
+            return True
+
+        return bool(pymailcheck.suggest(address))
```

Comment:
I'm not sure enforcing a bool return value is the best approach here. Devs may want the additional information that pymailcheck provides as suggestions, but we're forcing them to call that lib directly again after getting this result.

We may need to clarify what the desired code flow looks like. I assume these are intended primarily for form validation, so we can consider what messages, etc. will be user-facing out of this and adjust the API to match. Will there be a validator added in keg-elements that uses these methods?

Reply:
@guruofgentoo The original request was to enable a simple validation check prior to sending an e-mail, in order to replace similar functionality that was previously provided by mailgun. We can add form validation on to this, but it's beyond the originally-described scope.

In addition to matching the behavior inferred by the name, returning a `bool` on line 145 maintains parity with the return type on line 143. We can modify this method to return a more comprehensive result, but I'd then recommend we change to the name (to match the new behavior) and change the return type for an invalid address.

Reply:
That sounds reasonable, thanks.